src/pages/workspaces/workspace/Data.js

```diff
@@ -248,24 +232,47 @@ class WorkspaceDataContent extends Component {
             width, height: 500,
             rowCount: entities.length,
             columns: [
-              {
-                width: 150,
-                headerRenderer: () => h(SortableHeaderCell, {
-                  sort, field: 'name', onSort: v => this.setState({ sort: v })
-                }, [`${selectedDataType}_id`]),
-                cellRenderer: ({ rowIndex }) => renderDataCell(entities[rowIndex].name, namespace)
-              },
-              ..._.map(name => ({
-                width: 300,
-                headerRenderer: () => h(SortableHeaderCell, {
-                  sort, field: name, onSort: v => this.setState({ sort: v })
-                }, [name]),
-                cellRenderer: ({ rowIndex }) => {
-                  return renderDataCell(
-                    Utils.entityAttributeText(entities[rowIndex].attributes[name]), namespace
-                  )
+              (() => {
+                const thisWidth = theseColumnWidths['name'] || 150
+                return {
+                  width: thisWidth,
+                  headerRenderer: () => h(Resizable, {
+                    width: thisWidth, onWidthChange: delta => {
+                      theseColumnWidths['name'] = thisWidth + delta
+                      columnWidths[selectedDataType] = theseColumnWidths
+                      this.setState({ columnWidths })
```

Comment:
I'm sure there's a nice lodash way to do this, but I couldn't figure it out.

Reply:
There is! For reference, you can use `_.set` or `_.update` with a path array to do deep object 'modifications'.